bin/constants.js

```diff
@@ -0,0 +1,37 @@
+const NON_COMPILATION_ARGS = [
+	"init",
+	"migrate",
```

Comment:
Isn’t it a util folder where u can put this?

Reply:
I wasn't sure and thought utils dir is just for utility functions 😅, moved the file now 👍

Reply:
I think I'll move the `opencollective` file to utils as well, doing that in a separate issue. 😄